src/scrabble.js

```diff
@@ -1,5 +1,50 @@
-function scrabble() {
-
+function scrabble(word) {
+  const lettersAN = {
```

Comment:
I see you've put 'AN' to indicate capital letters, but does 'AN' actually mean anything? Perhaps a better name could be something like 'lettersCapital'.